README.md

```diff
@@ -114,21 +114,21 @@ Is user wants to manually connect to it then the screen shows the network name s
   <img src="/screenshots/send_normal_mode.gif" width="280" height="500"/>
 </p>
 
-For devices having Android version 7.0 and 7.1 user will be sent to settings to enable the wifi hotspot functionality. The user will then need to tap the device back button to come back to share.
+For devices having Android version 7.0 and 7.1 user will be sent to settings to enable the wifi hotspot functionality. The user will then need to tap the device back button to come back to skunkworks-crow.
 And for other device it will be enabled automatically.
 
 
 ### 3.3 Receive filled forms<a name="receive-filled-forms"></a>
 
-Tap on the “Receive” button shown at the bottom of the home screen to see a list of all wifi networks available in the nearby area which are started by only Share app.
+Tap on the “Receive” button shown at the bottom of the home screen to see a list of all wifi networks available in the nearby area which are started by only skunkworks-crow app.
 
 A receiver will need to connect to the network the sender has created. There are two ways to connect to the network:
 A QR code generated by the sender can be scanned by a receiver, it will get all the information required to connect to the sender (wifi name, port,  and password if the connection is protected).
 Tap on the wifi network available to connect and after successful connection enter port number to start receiving the forms.
 
 Connecting to the sender’s network will disable all the wifi network available so that no other network can interrupt the current connection. After receiving the forms it will restore the last saved state.
 
-For some devices, the user is not allowed to configure the wifi hotspot from the app. Currently for Android versions < 8, “ODK-SHARE-odk-share” is set as the default network name and for versions >= 8 it is AndroidShare-XXXX which is set by Android.
+For some devices, the user is not allowed to configure the wifi hotspot from the app. Currently for Android versions < 8, “ODK-skunkworks-crow-odk-skunkworks-crow” is set as the default network name and for versions >= 8 it is Androidskunkworks-crow-XXXX which is set by Android.
```

Comment:
here ODK-SHARE-odk-share as well as AndroidShare-XXXX should not be changed because nothing is changed in code so it will still start the hotspot with the same name. What do you say  about this? @shobhitagarwal1612

Reply:
Agreed @lakshyagupta21 

This change should be reverted from this PR and handled separately in a new PR. @lakshyagupta21 can you open a new issue?

Reply:
Sure

Reply:
Please Undo this change